dissect/executable/elf/elf.py

```diff
@@ -318,10 +318,14 @@ def dump_table(self) -> tuple[int, bytearray]:
 
 
 class StringTable(Section):
+    def __init__(self, fh: BinaryIO, idx: Optional[int] = None, c_elf: cstruct = c_elf_64):
+        super().__init__(fh, idx, c_elf)
+
+        self._get_string = lru_cache(self._get_string)
```

Comment:
```suggestion
        self._get_string = lru_cache(256)(self._get_string)
```
Let's put a sensible limit on this, I guess 256 is fine?

Reply:
Ah yeah. Yeah I doubt that too, but the data it would cache shouldn't we that large so I don't think it's too bad.